nixos/modules/services/networking/wireguard.nix

```diff
@@ -80,6 +80,15 @@ let
         description = "Commands called at the end of the interface setup.";
       };
 
+      preShutdown = mkOption {
+        example = literalExpression ''"''${pkgs.iproute2}/bin/ip netns del foo"'';
+        default = "";
+        type = with types; coercedTo (listOf str) (concatStringsSep "\n") lines;
```

Comment:
I think `lines` by itself already suffices.

Reply:
This is copied from the other options which do the same (but at different locations). 
Then this might be changed for all those options and not just this. I would like to do that in a seperate PR then, because it does not just affects the preShutdown option but is more a cleanup for all those options.